### ISSUE TITLE
Jade in Chapter6

### ISSUE THREAD

**Issue by spetroll** [deliciousinsights/brunch-guide#37]
Hi, having trouble completing Chapter 6.

First `jquery` is missing in your code snippets, either by using the npm global or requiring it in `application.js`.

After I added it, there seems to be a problem with jade: 

```
Uncaught Error: Cannot find module 'jade/runtime' from '___globals___' in jquery.js:9848
Uncaught Error: Cannot find module 'jade/runtime' from '___globals___' in list.jade:1
Uncaught ReferenceError: jade is not defined in list.jade:15
```

After fixing a few errors before, I'm not sure what exactly the problem is and how it can be resolved. 

Furthermore, [jade is being renamed to pug](https://github.com/pugjs/pug/issues/2184)..

**Comment by delameko** [comment]
You've probably already figured this out now (or abandoned it), but I'll leave this here in case others are having the same problem.

The solution was simply to remove that line.  So my global block looks like:

```
npm: {
  globals: {
    $: 'jquery'
  }
}
```

It appears Jade is automatically included so doesn't need to be manually added to the globals.

**Comment by shvaikalesh** [comment]
Closed by #49
